app/models/common.py

```diff
@@ -1,22 +1,18 @@
 import numpy as np
-
 from declarations import COLUMN_NAMES
 
 
 class PieceMove:
     from_pos: tuple[int, int]
     to_pos: tuple[int, int]
 
-    def __init__(self, board_move_split):
-        from_pos_board, to_pos_board = (
-            board_move_split[0],
-            board_move_split[1],
-        )
-        self.from_pos = self.position_converter(from_pos_board)
-        self.to_pos = self.position_converter(to_pos_board)
+    def __init__(self, board_move_split: list):
+        from_pos_board, to_pos_board = board_move_split[0], board_move_split[1]
+        self.from_pos = self.__position_converter(from_pos_board)
+        self.to_pos = self.__position_converter(to_pos_board)
 
     @staticmethod
-    def position_converter(pos_board: str) -> tuple[int, int]:
+    def __position_converter(pos_board: str) -> tuple[int, int]:
         x_pos_board, y_pos_board = (pos_board[0].upper(), pos_board[1])
 
         x_pos_array = np.where(COLUMN_NAMES == x_pos_board)[0][0]
```

Comment:
in this method, let's return x_pos_array (array of letters) first and y_pos_array (array of numbers) second, as letters always come first in chess.
this adjustment will impact all piece move validations, but it's best to correct this now than later.

Reply:
agreed